.eslintrc.json

```diff
@@ -25,7 +25,6 @@
   "rules": {
     "no-debugger": "off",
     "quotes": ["error", "double"],
-    "header/header": [2, "config/license-header.js"],
```

Comment:
Przeniosłem do osobnego configa, żeby to nie przeszkadzało przy uruchomieniu aplikacji